cli/tests/integrations/test_ssl.py

```diff
@@ -0,0 +1,91 @@
+import os
```

Comment:
Tests look good. What did we decide regarding the self signed cert from the admin router? We are missing a test for valid SSL and HTTPS url.